presentation/components/src/test/tree/controlled/TreeHooks.test.ts

```diff
@@ -33,6 +36,10 @@ describe("usePresentationNodeLoader", () => {
     pagingSize: 5,
   };
 
+  before(async () => {
+    await UiComponents.initialize(new I18N());
+  });
+

```

Comment:
This mutates global test environment. Either undo the changes after relevant tests complete, or perform this in the global test setup.

Reply:
Fixed, thanks